.gitignore

```diff
@@ -42,5 +42,6 @@ logs/
 lib/*
 !strongbox-security/strongbox-authentication-registry/src/test/resources/webapp/WEB-INF/lib/strongbox-example-authentication-provider-1.0-SNAPSHOT.jar
 !strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/test/resources/artifacts/properties-injector-1.7.jar
+!strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/test/resources/artifacts/slf4j-log4j12-1.7.26.jar
```

Comment:
@fuss86 : 

How are these files ending up here?

Reply:
They are proper jar files with classes inside. They are lightweight. They are used for testing purposes.